### PR TITLE
feat: implemented the profile page function

### PR DESCRIPTION
**Pull request by himanshukesarwani-dev**
- implemented the profile page function
- takes a img argument.
- returns the image string.

### CODE REVIEW

File: math.js [hunk 1]
```diff
@@ -62,4 +62,11 @@ function arithmetic(a,b)
 
 function pythagorasTheoremFunction(base, height){
     return Math.sqrt((base*base)+(height*height));
+}
+
+
+// code by himanshu.
+
```
Comment (jayshreejaiswal) [inline math.js:77]: can you please add documnetation

**Review by jayshreejaiswal — CHANGES_REQUESTED** [review]

File: math.js (hunk 1 above)
Comment (himanshukesarwani-dev) [reply math.js:77]: Added the documentation.

File: math.js [hunk 2]
```diff
@@ -62,4 +62,18 @@ function arithmetic(a,b)
 
 function pythagorasTheoremFunction(base, height){
     return Math.sqrt((base*base)+(height*height));
+}
+
+
+/**
+ * profile page function
```
Comment (jayshreejaiswal) [inline math.js:69]: can you please give info what is is this function about

**Review by jayshreejaiswal — CHANGES_REQUESTED** [review]

File: math.js (hunk 2 above)
Comment (himanshukesarwani-dev) [reply math.js:69]: Added the explaination.

**Review by SaiVasudha — APPROVED** [review]
LGFM

**Pull request merged**